zzzz_modular_occulus/code/game/objects/items/weapons/tools/mods/mod_types.dm

```diff
@@ -0,0 +1,134 @@
+/obj/item/weapon/tool_upgrade/augment/folding
+	name = "Collapsible tool kit"
+	desc = "This small toolkit adjusts a tool so that it can be collapsed down to a smaller size. This reduces the lifespawn of the tool, but makes it take up less space."
+	matter = list(MTERIAL_STEEL = 2, MATERIAL_PLASTIC = 2)
+	icon = 'zzzz_modular_occulus/icons/obj/tool_upgrades.dmi'
+	icon_state = "foldkit"
+	price_tag = 120
+
+/obj/item/weapon/tool_upgrade/augment/folding/New()
+	..()
+	var/datum/component/item_upgrade/I = AddComponent(/datum/component/item_upgrade)
+	I.tool_upgrades = list(
+	UPGRADE_DEGRADATION_MULT = 2,
+	UPGRADE_BULK = -1
+	)
+	I.prefix = "Collapsible"
```

Comment:
Does this append a capitalized 'Collapsible' to the item name? eg. 'the sanctified edged wrenching Collapsible shovel'

Reply:
fixed